pom.xml

```diff
@@ -82,22 +83,14 @@
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
-      <version>4.12</version>
```

Comment:
Version is managed through dependency management: https://github.com/querydsl/querydsl/pull/2729/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R149